src/CalcManager/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_library(CalcManager
+	CalculatorHistory.cpp
+	CalculatorManager.cpp
+	ExpressionCommand.cpp
+	pch.cpp
```

Comment:
I'm at odds when it comes to this file. It's not required for the `CalcManager` library at all, it does not provide any symbols, but then again it makes sense to test its compilability. It would probably be best to leave it be for now, though maybe with a comment explaining the purpose of this file being here. I will try to mimic #436 with cmake in the future and probably address this issue.